### PR TITLE
Swaggerhub

### PR DESCRIPTION
**Pull request by KaiWissel**

### CODE REVIEW

File: OpenAPI/swagger.json [hunk 1]
```diff
@@ -0,0 +1,1600 @@
...
+    {
+      "name": "mail"
+    }
+  ],
+  "schemes": ["https", "http"],
+  "paths": {
+    "/mail": {
+      "post": {
+        "tags": ["mail"],
+        "summary": "Send a message to a number of users",
```
Comment (KaiWissel) [inline OpenAPI/swagger.json:55]: Das ist ein Kommentar

Comment (JEK58) [reply OpenAPI/swagger.json:55]: This is a reply

File: OpenAPI/swagger.json [hunk 2]
```diff
@@ -1158,327 +1234,367 @@
...
-        "flightType" : {
-          "type" : "string",
-          "enum" : [ "JOJO", "Dreieck", "FAI Dreieck" ]
+        "flightPoints": {
+          "type": "number"
         },
-        "flightStatus" : {
-          "type" : "string",
-          "enum" : [ "Nicht in Wertung", "In Wertung", "Flugbuch", "In Bearbeitung" ]
+        "fixes": {
```
Comment (JEK58) [inline OpenAPI/swagger.json:1283]: Include fixes in flight model?

File: OpenAPI/swagger.json [hunk 3]
```diff
@@ -1158,327 +1234,367 @@
...
+        "airbuddies": {
+          "type": "array",
+          "items": {
+            "flightId": "string",
+            "pilotName": "string"
+          }
         },
-        "flightDistance" : {
-          "type" : "number"
+        "takeoffTime": {
```
Comment (JEK58) [inline OpenAPI/swagger.json:1303]: Add takeoff time to Flight model

File: OpenAPI/swagger.json [hunk 4]
```diff
@@ -1158,327 +1234,367 @@
...
+          "type": "string"
         },
-        "flightPoints" : {
-          "type" : "number"
+        "landingTime": {
+          "type": "string"
```
Comment (JEK58) [inline OpenAPI/swagger.json:1307]: Add landing time to Flight model

File: OpenAPI/swagger.json [hunk 5]
```diff
@@ -1158,327 +1234,367 @@
...
         },
-        "submitted" : {
-          "type" : "string",
-          "format" : "date-time"
+        "takeOff": {
```
Comment (JEK58) [inline OpenAPI/swagger.json:1309]: Add takeoff to Flight model

File: OpenAPI/swagger.json [hunk 6]
```diff
@@ -1158,327 +1234,367 @@
...
-        "tshirtSize" : {
-          "type" : "string",
-          "enum" : [ "S", "M", "L", "XL", "XXL" ]
+        "tshirtSize": {
+          "type": "string",
+          "enum": ["S", "M", "L", "XL", "XXL"]
         },
-        "aircraft" : {
-          "$ref" : "#/definitions/Aircraft"
+        "aircrafts": {
```
Comment (JEK58) [inline OpenAPI/swagger.json:1412]: Allow multiple aircrafts in profile

**Review by JEK58 — CHANGES_REQUESTED** [review]

File: OpenAPI/swagger.json (hunk 2 above)
Comment (KaiWissel) [reply OpenAPI/swagger.json:1283]: Done

File: OpenAPI/swagger.json (hunk 3 above)
Comment (KaiWissel) [reply OpenAPI/swagger.json:1303]: Flight model extended by "fixes" object with timestamps
First timestamp == takeoffTime

File: OpenAPI/swagger.json (hunk 4 above)
Comment (KaiWissel) [reply OpenAPI/swagger.json:1307]: Flight model extended by "fixes" object with timestamps
Last timestamp == landingTime

File: OpenAPI/swagger.json (hunk 5 above)
Comment (KaiWissel) [reply OpenAPI/swagger.json:1309]: Flight model extended by "fixes" object with timestamps
First timestamp == takeoffTime

File: OpenAPI/swagger.json (hunk 6 above)
Comment (KaiWissel) [reply OpenAPI/swagger.json]: done

**Pull request closed without merging**